Source/carthageTests/VersionCommandSpec.swift

```diff
@@ -0,0 +1,62 @@
+import Quick
+import Nimble
+import CarthageKit
+import Commandant
+import Cocoa
+
+import carthage
+
+private class MockPrinter: Printer {
```

Comment:
Probably something that'll happen when it gets moved to it's own file so that I can share the MockPrinter with other spec files.